### ISSUE TITLE
Call to undefined method Roave\BetterReflection\Util\FindReflectionOnLine::buildDefaultFinder()

### ISSUE THREAD

**Issue by bendavies** [thecodingmachine/phpstan-strict-rules#12]
Hey,

Can't get your rules working at the moment.
Reproduction:

`composer require --dev thecodingmachine/phpstan-strict-rules roave/better-reflection:"dev-master as 2.0.0"`

...add your rules...

```
./vendor/bin/phpstan analyse -c phpstan.neon -l max src
 ------ -----------------------------------------------------------------------------------------------------------------
  Line   src/Serializer/Handler/DateTimeHandler.php
 ------ -----------------------------------------------------------------------------------------------------------------
         Internal error: Call to undefined method Roave\BetterReflection\Util\FindReflectionOnLine::buildDefaultFinder()
 ------ -----------------------------------------------------------------------------------------------------------------
```

Thanks!

**Comment by moufmouf** [comment]
Damn! better-reflection is a moving target as they are actively working towards 2.0.0.
I'm eagerly waiting for them to have a stable version so I can tag this library as well.

Meanwhile, I've fixed the problem in #13. Let me know if this works for you!

**Comment by bendavies** [comment]
awesome, thanks for the quick fix!
